index.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const logger = require('morgan');
+const helmet = require('helmet');
+const cors = require('cors');
+const knex = require('knex');
+
+const server = express();
+
+
+server.use(express.json(), logger('combined'), cors(), helmet());
+
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
+
+server.get('/', (req, res) => {
+    res.send('Test for root endpoint!')
+});
+
+//Projects Post
+server.post('/api/projects',(req, res) => {
+    const {name, description, completed} = req.body;
+    const project = {name, description, completed};
+    if (!project) res.status(400).json({ error: 'Please provide a project' });
+      db.insert(project)
+        .into('projects')
+            .then(ids => res.status(201).json( ids))
+                .catch(err => res.status(500).json({ error: 'Project cannot be saved' }));
+  });
+
+//GET Project By ID
+server.get('/projects/:id', (req, res) => {
+    const { id } =  req.params.id;
+    db('projects')
+    .select()
+            .where({ id })
+                .then(projects => {
+                    if(project) {
+                        res.status(200).json(projects);
```

Comment:
There seems to be nothing called `project` in this scope, so you're never going to get the project back this way. Try to be mindful and consistent with what you name your variables!